x/clock/types/params_test.go

```diff
@@ -10,35 +10,54 @@ import (
 
 func TestParamsValidate(t *testing.T) {
 	testCases := []struct {
-		name     string
-		params   types.Params
-		expError bool
+		name    string
+		params  types.Params
+		success bool
 	}{
-		{"default", types.DefaultParams(), false},
 		{
-			"valid: no contracts, enough gas",
-			types.NewParams([]string(nil), 100_000),
-			false,
+			"Success - Default",
+			types.DefaultParams(),
+			true,
 		},
 		{
-			"invalid: address malformed",
-			types.NewParams([]string{"invalid address"}, 100_000),
+			"Success - Meets min Gas",
+			types.NewParams(100_000),
 			true,
 		},
 		{
-			"invalid: not enough gas",
-			types.NewParams([]string(nil), 1),
+			"Success - Meets min Gas",
+			types.NewParams(500_000),
 			true,
 		},
+		{
+			"Fail - Not Enough Gas",
+			types.NewParams(1),
+			false,
+		},
+		{
+			"Fail - Not Enough Gas",
+			types.NewParams(100),
+			false,
+		},
+		{
+			"Fail - Not Enough Gas",
+			types.NewParams(1_000),
+			false,
+		},
+		{
+			"Fail - Not Enough Gas",
```

Comment:
Do / did we get a test for executing more than the block gas and ensuring they still go through?

block gas of 1m
15x contracts at 100k gas should still execute, even though they are >1m

Reply:
We do have a performance test which runs 1000 contracts which utilize about 70k gas each in the abci_test.go (Line 2060.

Reply:
perfect